src/ui/components/button/Button.module.css

```diff
@@ -0,0 +1,16 @@
+.button {
+  padding: 10px 16px;
+  font-size: 14px;
```

Comment:
Use variables (basically it should be dependent on the design guidelines, if the titles are supposed to be 20px then that should be a --font-size-title variable and so on